app/api/events/past/route.ts

```diff
@@ -0,0 +1,20 @@
+import { hyphenateUrl } from "app/api/hyphenateUrl";
+import { NextRequest } from "next/server";
+import client from "../../../../tina/__generated__/client";
+import { getEventsWithClient } from "../getEvents";
+
+export async function GET(req: NextRequest) {
+  let presenterName = req.nextUrl.searchParams.get("presenterName");
+  if (!presenterName)
+    return new Response("presenterName is required", { status: 400 });
```

Comment:
```suggestion
  if (!presenterName) {
    return new Response("presenterName is required", { status: 400 });
  }
  
```
braces for readability

Reply:
Done - I've updated the if statements in this file to use braces (I removed the 400 so this suggestion is no longer relevant, see reply https://github.com/SSWConsulting/SSW.Website/pull/2828#issuecomment-2232047081)